contracts/test/strategies/dripper.js

```diff
@@ -0,0 +1,128 @@
+const { expect } = require("chai");
+const { defaultFixture } = require("../_fixture");
+
+const { loadFixture, usdtUnits, advanceTime } = require("../helpers");
+
+describe.only("Dripper", async () => {
```

Comment:
.only

Reply:
Thanks!

Reply:
One day we need a GitHub action for that. :D

Reply:
Yeah, and also one that checks that no deploy after nr.6 has a `forceDeploy` set to `true`